src/main/kotlin/org/sol4k/instruction/CreateAssociatedTokenAccountInstruction.kt

```diff
@@ -13,7 +14,7 @@ class CreateAssociatedTokenAccountInstruction(
     mint: PublicKey,
 ) : Instruction {
 
-    override val data: ByteArray = ByteArray(0)
+    override val data: ByteArray = byteArrayOf(0)
```

Comment:
a bug fix here

Reply:
What was the bug?

Reply:
`ByteArray(0)` means an empty byte array for which the associated-token-account has `0x0` as data